starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,103 @@
+          <div class="background-img">
+            <h1>Apple Pie</h1>
+          </div>
+          <p class="paragraph">
+            This was my grandmother's apple pie recipe. I have never seen another
+            one quite like it. It will always be my favorite and has won me
+            several first place prizes in local competitions. I hope it becomes
+            one of your favorites as well!
+          </p>
+          
+          <p class="no-line-break">
+            1h30min <img src="./images/circular-clock.png" alt="Time" /> 8
+            servings <img src="./images/pie-chart.png" alt="pie" /> 512cals
+            <img src="./images/bars (1).png" alt="bars" />
+          </p>
+        </section>
+        <hr />
+        <section>
+          <h2>Ingredients</h2>
+          <ul>
+            <li>
+              - 1 recipe pastry for a 9 inch double crust pie
+            </li>
+            <li>
+              - 1/2 cup unsalted butter
+            </li>
+            <li>
+              - 3 tablespoons all-purpose flour
+            </li>
+            <li>
+              - 1/4 cup water
+            </li>
+            <li>
+              - 3 tablespoons all-purpose flour
+            </li>
+            <li>
+              - 1/4 cup water
+            </li>
+            <li>
+              - 1/2 cup white sugar
+            </li>
+            <li>
+              - 1/2 cup packed brown sugar
+            </li>
+            <li>
+              - 8 Granny Smith apples - peeled, cored and sliced
+            </li>
+          </ul>
+          <div class="flex-row">
+            <div class="items">
+              <img src="./images/circular-clock.png" alt="Time" />
+            </div>
+            <div class="items border-right">
+              <p>Prep <br /><strong>30</strong> m</p>
+            </div>
+            <div class="items border-right">
+              <p>Cook <br /><strong>1</strong> h</p>
+            </div>
+            <div class="items">
+              <p>
+                Ready in
+                <br /><strong>1</strong> h <strong>30</strong> m
+              </p>
+            </div>
+          </div>
+          <hr />
+        </section>
+        <h2>
+          Directions
+        </h2>
+        <p>
+          1) Preheat oven to 425 degrees F (220 degrees C). Melt the butter
+          in a saucepan. Stir in flour to form a paste. Add water, white sugar and
+          brown sugar, and bring to a boil. Reduce temperature and let simmer. Place
+        </p>
+        <p>
+          2) the bottom crust in your pan. Fill with apples, mounded slightly. Cover
+          with a lattice work crust. Gently pour the sugar and butter liquid over
+          the crust. Pour slowly so that it does not run off.
+        </p>
+        <p>
+          3) Bake 15 minutes in the
+        preheated oven. Reduce the temperature to 350 degrees F (175 degrees C).
+        Continue baking for 35 to 45 minutes, until apples are soft.
+      </p>
```

Comment:
Aqui deveria ser uma lista ordenada ("**ol**") ao invés de paragrafos, algo como:

<"ol">
   <"li"> Preheat oven to 425... <"/li">
   <"li"> the bottom crust... <"/li">
   <"li"> Bake 15 minutes in... <"/li">
<"ol">